src/ui/server/controllers/insurance/export-contract/map-submitted-data/agent-service-charge/index.ts

```diff
@@ -0,0 +1,29 @@
+import FIELD_IDS from '../../../../../constants/field-ids/insurance/export-contract';
+import { objectHasProperty } from '../../../../../helpers/object';
+import { RequestBody } from '../../../../../../types';
+
+const {
+  AGENT_CHARGES: { FIXED_SUM_AMOUNT, CHARGE_PERCENTAGE },
+} = FIELD_IDS;
+
+/**
+ * mapSubmittedData
+ * Map agent service charge fields
+ * @param {Express.Request.body} formBody
```

Comment:
requestBody?

Reply:
Just noticed this. We have 37 instance of this, would like to update in one go in a new PR.